src/fw/drivers/pmic/npm1300.c

```diff
@@ -166,6 +169,8 @@ bool pmic_init(void) {
   ok &= prv_write_register(PmicRegisters_GPIOS_GPIOMODE1, PmicRegisters_GPIOS_GPIOMODE__GPOIRQ);
   ok &= prv_write_register(PmicRegisters_GPIOS_GPIOOPENDRAIN1, 0);
 
+  ok &= prv_write_register(PmicRegisters_SHIP_SHPHLDCONFIG, PmicRegisters_SHIP_SHPHLDCONFIG__SHPHLDTIM_96MS);
```

Comment:
You need to trigger TASKSHPHLDCONFIGSTROBE after this.

Reply:
fixed